src/common/cm/cm_load.cpp

```diff
@@ -764,6 +774,12 @@ void CM_LoadMap(Str::StringRef name)
 		Sys::Drop("Could not load %s", mapFile.c_str());
 	}
 
+	std::string externalEntities = FS::PakPath::ReadFile( "maps/" + name + ".ent", err );
+	if ( err )
```

Comment:
We should drop if the error is something other than file not found. Here's an example from CG_ParseWeaponAnimationFile
```
if ( err )
	{
		const std::error_code notFound(Util::ordinal(FS::filesystem_error::no_such_file), FS::filesystem_category());
		// Some weapons e.g. dretch attack actually don't have animations
		if ( err != notFound )
		{
			Log::Warn( "couldn't read weapon animation file '%s': %s", filename, err.message() );
		}
		return false;
	}
```

Reply:
Why can this happen? It seems that the following errors are prevented by trying to read the .bsp first:

- `no_filesystem_error`
- `invalid_filename`
- `no_such_directory`
- `wrong_pak_checksum`
- `missing_dependency`

Reply:
Any error that has to do with actually reading the file. Note that most errors of this type are returned using the standard library's system error category rather than the custom `filesystem_error` one.